cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -479,3 +480,58 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+@pytest.mark.parametrize('form_data_vm_obj_single_datastore', [['nfs', 'nfs', rhel7_minimal]],
+                         indirect=True)
+def test_migration_policy_tag(request, appliance, v2v_providers, host_creds, conversion_tags,
+                              form_data_vm_obj_single_datastore, soft_assert):
+    """Test policy to prevent source VM from starting if migration is complete"""
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    # form_data_vm_obj_single_datastore has form_data at [0]
```

Comment:
Optional:
:( still, I am in confusion why we are going like this. I think `dict` is the best option in such a case. It will help you to get data for specific `key`